### PR TITLE
Azure Resouces - adding tags

### LINKED ISSUE #748: ♻️ Infra - Add Cost-Category tag to all resources
**Describe the bug**
Not exactly a bug, but all resources used in SSW need to be tagged with a cost-category of 'dev/test', 'value' or 'core'
dev/test - for things you are working on
value - for things that aren't required but are important
core - for things that are core to the business.  I.E. website

As per https://ssw.com.au/rules/azure-budgets/

1. Please update the infra deployment for this so that my Azure Cost Analysis weekly journey is easy 😊
2. Get 'Checked by Warwick'

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixes #748 

#### Description:
Implemented a tagging for Azure resources, utilizing 'dev/test,' 'core,' and 'value' tags to effectively categorize and track their associated costs.

### CODE REVIEW

**Review by leahy268 — APPROVED** [review]
Looks ok to me although I'm not completely familiar with the Infra in the repo

File: infra/main.bicep [hunk 1]
```diff
@@ -30,31 +30,36 @@ param dockerImage string = 'sswwebsite'
...
+  'cost-category': 'value'
 }
 
 module acr 'acr.bicep' = {
   name: 'acr-${now}'
   params: {
     projectName: projectName
     location: location
-    tags: tags
+    tags: core
```
Comment (wicksipedia) [inline infra/main.bicep]: if this is always `core` - then it should be set in the module and not determined outside in the `main` file

File: infra/main.bicep [hunk 2]
```diff
@@ -30,31 +30,36 @@ param dockerImage string = 'sswwebsite'
...
   }
 }
 module keyVault 'keyVault.bicep' = {
   name:'keyVault-${now}'
   params: {
     projectName: projectName
     location: location
+    tags:core
```
Comment (wicksipedia) [inline infra/main.bicep]: if this is always `core` - then it should be set in the module and not determined outside in the `main` file

**Review by wicksipedia — APPROVED** [review]
Approved with suggestions

File: infra/main.bicep (hunk 1 above)
Comment (amankumarrr) [reply infra/main.bicep]: Done - yes it makes sense

File: infra/main.bicep (hunk 2 above)
Comment (amankumarrr) [reply infra/main.bicep]: Done - yes it makes sense

**Review by wicksipedia — APPROVED** [review]

**Pull request merged**